core/src/main/scala/org/apache/spark/deploy/mesos/ui/DriverPage.scala

```diff
@@ -50,7 +50,7 @@ private[ui] class DriverPage(parent: MesosClusterUI) extends WebUIPage("driver")
     val driverDescription = Iterable.apply(driverState.description)
     val submissionState = Iterable.apply(driverState.submissionState)
     val command = Iterable.apply(driverState.description.command)
-    val schedulerProperties = Iterable.apply(driverState.description.schedulerProperties)
+    val schedulerProperties = Iterable.apply(driverState.description.sc.getAll.toMap)
```

Comment:
`conf` is still namespaced under `driverState`, so I think it's still just as clear that these are driver properties.  In fact, maybe even moreso, since "scheduler" is ambiguous, as both the dispatcher and the driver are schedulers.